ONSdigital/sixteens
# Fix - Error when starting

### What
1. There was an error when running `./run.sh`. The onchange path was incorrect. This change is to use the bin versions of commands instead of going to path. The path is an implementation detail of the module and should be avoided.
1. Update dependencies

### How to review
1. Ensure that `./run.sh` starts without any errors
1. Ensure that `npm run build` runs successfully

### Who can review
Anyone
